tests/Domain/Sniffs/ForbiddenSetterMethods/ForbiddenSetterMethodsTest.php

```diff
@@ -28,9 +27,13 @@ public function prepareFixtureWithSniff(string $sniffClassName, string $fixtureF
         $sniffs = [self::getFilePathFromClass($sniffClassName)];
 
         $config = new Config();
+        $config->standards = [];
```

Comment:
This line throw an error in phpstan

>  Access to an undefined property PHP_CodeSniffer\Config::$standards

Could you check why ?

Reply:
It is not a property defined, it is accessed via magic setter :)